### DISCUSSION TITLE
Server-阿里云 ECS | Zi-Zi's Journey

### DISCUSSION THREAD

**Discussion by giscus** [GZ-Metal-Cell/GZ-Metal-Cell.github.io#325] [Announcements]
# Server-阿里云 ECS | Zi-Zi's Journey

瞎折腾之将博客部署至阿里云服务器！

https://gz-metal-cell.github.io/2024/01/29/Server-%E9%98%BF%E9%87%8C%E4%BA%91%20ECS/

**Comment by promefire** [comment]
😭太细心了Metal哥😭全文所有图片都没有泄露IP😭

**Reply by GZ-Metal-Cell** [reply]
😭ip是121.41.23.127😭快到期了😭
